package.json

```diff
@@ -6,7 +6,8 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test:unit": "vue-cli-service test:unit --testPathPattern"
+    "test:unit": "vue-cli-service test:unit --testPathPattern",
+    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
```

Comment:
Could you give a quick tech demo on how to use this? Thx!

Reply:
sure, post-stand up on Monday?

Reply:
That would be great? Or maybe right before sprint retro (like we used to do)? Ask Shahriar.